### DISCUSSION TITLE
How to create a gdg base with ZOAU?

### DISCUSSION THREAD

**Discussion by FALLAI-Denis** [IBM/zoau-samples#70] [Q&A]
Hi,

ZOAU plans to create VSAM KSDS, RRDS, ESDS files using the dtouch command with the -t option.

We would like to be able to create GDG bases which are another type of VSAM file, but dtouch does not plan to support this type of file.

Is there another way to create the base of a GDG file, specifying at least the number of generations to manage?

Then it will be necessary to be able to create a new generation, and to access the existing generations in relative mode:
- base.gdg(0)
- base.gdg(+1)
- base.gdg(-1)
- etc.

Thanks.

**Comment by AnthonyGiorgio** [comment]
ZOAU has plans to support GDG in the v1.3 release stream.  v1.3.0 already has support for GDGs in some limited cases.
